<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
		background-color: chocolate;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
  </property>
  <widget class="QLabel" name="sllabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: chocolate;
		border: 1px solid chocolate;
		color: black;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>Single LInked List</string>
   </property>
  </widget>
  <widget class="QLabel" name="bstlabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: chocolate;
		border: 1px solid chocolate;
		color: black;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>Binary Search Tree</string>
   </property>
  </widget>
  <widget class="QLabel" name="sllabel_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: chocolate;
		border: 1px solid chocolate;
		color: black;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>      Created LIst</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete2button_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>430</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
		background-color: red;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="datainput_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deletebutton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
		background-color: red;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editbutton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
		background-color: GreenYellow;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>Insert Pos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insertbutton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
		background-color: yellow;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>INSERT</string>
   </property>
  </widget>
  <widget class="QLabel" name="sllabel_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: chocolate;
		border: 1px solid chocolate;
		color: black;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>      Created Tree</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="datainput">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="outputlabel_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: chocolate;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sllabel_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: chocolate;
		border: 1px solid chocolate;
		color: black;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>Got</string>
   </property>
  </widget>
  <widget class="QComboBox" name="structurescb">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="posl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="getbutton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
		background-color: navy;
		border: 1px solid black;
		color: white;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>GET</string>
   </property>
  </widget>
  <widget class="QLabel" name="datalabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
		background-color: chocolate;
		border: 1px solid chocolate;
		color: black;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>        DATA STRUCTURES</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insert2button_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
		background-color: GreenYellow;
		border: 1px solid black;
		color: black;
		padding: 5px; 	
}</string>
   </property>
   <property name="text">
    <string>INSERT</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="display_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="display_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
